features/authorization.md

```diff
@@ -190,15 +191,22 @@ Istio Authorization provides access control for workloads in the mesh at the nam
 
 - [ ] Latency, throughput, and scalability are quantified and documented on
 	istio.io. 
+- [X] Stability and Performance tests to test the policies addition and deletion in bulk.
+
+**Tests**
+
+- [X] Test all the possible fields from CRD to imporve the test coverage.
+= [X] Test the performance and stabiltity of control plane while applying and deleting the policies in bulk.
+- [X] Fix issues in multicluster scenarios.
 
 **Bugs**
 
-- [ ] Feature has no known major issues. 
+- [X] Feature has no known major issues. 
 
 **Approvals**
 
-- [ ] The appropriate work group(s) have reviewed and approved promotion of the feature.
-- [ ] The supportability review panel has reviewed promotion of the feature.  
+- [X] The appropriate work group(s) have reviewed and approved promotion of the feature.
+- [X] The supportability review panel has reviewed promotion of the feature.  
 - [ ] The TOC has reviewed and approved promotion of the feature as part of the
```

Comment:
I think you can mark this complete as the TOC will have approved promotion.

Reply:
updated, thanks